sdk/attestation/azure-security-attestation/azure/security/attestation/_configuration.py

```diff
@@ -24,28 +20,26 @@ class AttestationClientConfiguration(Configuration):
     Note that all parameters used to create this instance are saved as instance
     attributes.
 
-    :param credential: Credential needed for the client to connect to Azure.
-    :type credential: ~azure.core.credentials.TokenCredential
-    :param instance_url: The attestation instance base URI, for example https://mytenant.attest.azure.net.
-    :type instance_url: str
-    :keyword validation_options: Optional token validation options.
-    :type validation_options: TokenValidationOptions
+    :keyword bool validate_token: if True, validate the token, otherwise return the token unvalidated.
+    :keyword validation_callback: Function callback to allow clients to perform custom validation of the token.
+        if the token is invalid, the `validation_callback` function should throw
+        an exception.
+    :paramtype validation_callback: Callable[[AttestationToken, AttestationSigner], None]
+    :keyword bool validate_signature: if True, validate the signature of the token being validated.
+    :keyword bool validate_expiration: If True, validate the expiration time of the token being validated.
+    :keyword str issuer: Expected issuer, used if validate_issuer is true.
+    :keyword float validation_slack: Slack time for validation - tolerance applied
+        to help account for clock drift between the issuer and the current machine.
+    :keyword bool validate_issuer: If True, validate that the issuer of the token matches the expected issuer.
+    :keyword bool validate_not_before_time: If true, validate the "Not Before" time in the token.
     """
 
     def __init__(
```

Comment:
Lines 37-40 can become:
```suggestion
    def __init__(self, **kwargs):
        # type: (**Any) -> None
```